bin/improver-weighted-blending

```diff
@@ -122,82 +200,93 @@ def main():
                                 'non linear weights will be. '
                                 'A value of 1 implies equal weighting. If not '
                                 'set, a default value of cval=0.85 is set.')
-    parser.add_argument('--coord_adj', metavar='COORD_ADJUSTMENT_FUNCTION',
-                        type=str,
-                        help='Function to apply to the coordinate after the '
-                             'blending has been applied.')
-    parser.add_argument('--wts_redistrib_method',
-                        metavar='METHOD_TO_REDISTRIBUTE_WEIGHTS',
-                        default='evenly',
-                        help='The method to use when redistributing weights '
-                             'in cases where forecasts are missing. Options: '
-                             '"evenly": redistribute weights evenly between '
-                             'the forecasts that are available. '
-                             '"proportional": redistribute weights using the '
-                             'original weighting function.')
-    parser.add_argument('--cycletime', metavar='CYCLETIME', type=str,
-                        help='The forecast_reference_time to be used after '
-                             'blending has been applied in the format '
-                             'YYYYMMDDTHHMMZ.')
-    parser.add_argument('--coords_for_bounds_removal',
-                        metavar='COORDS_FOR_BOUNDS_REMOVAL', nargs="+",
-                        help='The forecast_reference_time to be used after '
-                             'blending has been applied in the format '
-                             'YYYYMMDDTHHMMZ.')
+
+    wts_dict = parser.add_argument_group('dict weights options',
+                                         'Options for linear weights to be '
+                                         'calculated based on parameters '
+                                         'read from a json file dict')
+    wts_dict.add_argument('--wts_dict', metavar='WEIGHTS_DICTIONARY',
```

Comment:
I assume that somewhere in the code the format of this dictionary is described in a docstring somewhere. Could we point to it?